crates/sui-json-rpc-types/src/lib.rs

```diff
@@ -1920,6 +1921,74 @@ pub enum SuiEvent {
     Checkpoint(CheckpointSequenceNumber),
 }
 
+impl TryFrom<SuiEvent> for Event {
```

Comment:
Where is this used?  Not sure when we would want to convert from SuiEvent back to Event....

Reply:
it is used here https://github.com/MystenLabs/sui/blob/2c00dcf58551d174184dfe52ef4cec0048ba7d6d/crates/sui-rosetta/src/unit_tests/balance_changing_tx_tests.rs#L263-L269

currently it's only used in test, but there are plan to make rosetta talks to a fullnode RPC server instead of embedding a fullnode, which will need this conversion. 
I think we need to revisit the RPC serialisation implementation, keeping 2 sets of models is becoming hard to maintain.

Reply:
Yeah I agree we should visit the object model.  I think it's a bit clumsy too.  Maybe we can reuse Event somehow.